docs/siem/detections/api/exceptions/exceptions-api-overview.asciidoc

```diff
@@ -0,0 +1,52 @@
+[[exceptions-api-overview]]
+== Exceptions API
+
+Exceptions are associated with detection and endpoint rules, and are used to
+prevent a rule from generating an alert from incoming events even when the
+rule's other criteria are met. They can be used to reduce the number of false
+positives, and to prevent trusted processes and network activity from
+generating unnecessary alerts.
+
+Exceptions are made up of:
+
+* *Exception containers*: A container for related exceptions. In general, a
+single exception container contains all the exception items relevant for
+a subset of rules. For example, a container can be used to group together
+network-related exceptions that are relevant for a large number of network
+rules. The container can then be associated with all the relevant rules.
+* *Exception items*: The query (fields, values, and logic) used to prevent
+rules from generating alerts. When an exception item's query evaluates to
+`true`, the rule does not generate an alert.
+
+For detection rules, you can also use <<lists-api-overview, lists>> to define
+rule exceptions. A list holds multiple values of the same {es} data type, such
+as IP addresses, which are used to determine when an exception prevents an
+alert from being generated.
+
+IMPORTANT: You cannot use lists with endpoint rule exceptions.
```

Comment:
You also can't use `operator: "excluded"` with endpoint exceptions, only `operator: "included"`